tests/integration/template_min_case_e3sm_diags_tc_analysis_parallel.cfg

```diff
@@ -0,0 +1,36 @@
+[default]
```

Comment:
New min-case test where E3SM Diags runs in parallel to generate `tc_analysis` plots for two year sets.